eslint/Tests/EslintUnitTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\User\Block\Role\Tab;
+
+use function PHPUnit\Framework\assertStringContainsString;
+
+class EslintUnitTest extends \PHPUnit\Framework\TestCase
+{
+    const command = '../node_modules/.bin/eslint -c eslint/.eslintrc --rulesdir eslint/rules ';
+
+    public function testAndSelf() {
+        $inputFile = 'eslint/Tests/jquery-andSelf.js';
+        $expectedMessage = 'jQuery.andSelf() removed, use jQuery.addBack()';
+        exec(
+            self::command . $inputFile,
+            $output,
+            $result
+        );
+
+        assertStringContainsString($expectedMessage, $output[2]);
```

Comment:
Please use the inherited methods instead of importing a function
```suggestion
        $this->assertStringContainsString($expectedMessage, $output[2]);
```